### ISSUE TITLE
Expected Graph Structure for Field assignment 

### ISSUE THREAD

**Issue by popthink** [plume-oss/plume#105]
Currently, Plume makes a graph for field assignment like as following.

someField = value
```
CALL(<op>.assignment)
      -AST->FieldIdentifier ("someField")
      -AST->Identifier("value")
```
But Joern expects the graph as following for such code .
```
Call(<op>.assignment)
  -AST->Call(<op>.fieldAccess) -AST-> Identifier("this") 
                          |-AST-> FieldIdentifier("someField")
  -AST->Identifier("value")
```

**Comment by popthink** [comment]
In case of Static Member assignment, maybe have to figure out which one is ... for left.
SomeClass.staticField = "A"
```
Call(<op>.assignment)
  -AST->Call(<op>.fieldAccess) -AST-> Identifier("SomeClass") or  Type_Ref("SomeClass")?
                                |-AST-> FieldIdentifier("staticField")
  -AST->Literal("A")
```

In case of instance Member assignment,
v1.instancePubField = "A"
```
Call(<op>.assignment)
  -AST->Call(<op>.fieldAccess) -AST-> Identifier("v1")
                                |-AST-> FieldIdentifier("staticField")
  -AST->Literal("A")
```

**Comment by popthink** [comment]
sub-issue of #93

**Comment by DavidBakerEffendi** [comment]
@popthink Can this issue be closed by #115?

**Comment by popthink** [comment]
Sure. Closed.
